subsys/fs/littlefs_fs.c

```diff
@@ -184,6 +195,32 @@ static int lfs_api_erase(const struct lfs_config *c, lfs_block_t block)
 	return errno_to_lfs(rc);
 }
 
+#if IS_ENABLED(CONFIG_FS_LITTLEFS_BLK_DEV)
```

Comment:
Change this to `#ifdef` or `#if defined()`; the `IS_ENABLED` is not needed here completely.